sdk/scheduler/src/main/java/com/mesosphere/sdk/config/DefaultConfigurationUpdater.java

```diff
@@ -30,6 +28,7 @@
 public class DefaultConfigurationUpdater implements ConfigurationUpdater<ServiceSpec> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultConfigurationUpdater.class);
+    private static final String USER = "root";
```

Comment:
Thanks for pointing this out @mrbrowning. I would argue for sticking (and possibly renaming) the one in `DefaultConfigurationUpdater` instead of the one in `SchedulerUtils`. With the current implementation (using the one in `SchedulerUtils`), we end up importing the class **only** for this variable (and not the other utility methods). How do we handle other such constants?